src/Application/Features/Reviewer/Command/UpdateReviewer/UpdateReviewerCommandValidator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using MRA.Jobs.Application.Contracts.Reviewer.Command;
+
+namespace MRA.Jobs.Application.Features.Reviewer.Command.UpdateReviewer;
+
+public class UpdateReviewerCommandValidator : AbstractValidator<UpdateReviewerCommand>
+{
+    public UpdateReviewerCommandValidator()
+    {
+        RuleFor(r => r.Id).NotEmpty();
```

Comment:
add other properties to command and add valiation

Reply:
ok